syntax="proto2";

package bssid_location;

message Location {
    optional int64 lat = 1;
    optional int64 lon = 2;
    optional int64 unk1 = 3; 
    optional int64 unk2 = 4; 
    optional int64 unk3 = 5; 
    optional int64 unk4 = 6; 
    optional int64 unk5 = 7; 
    optional int64 unk6 = 8; 
    optional int64 unk7 = 9; 
    optional int64 unk8 = 10; 
    optional int64 unk9 = 11; 
    optional int64 unk10 = 12; 
    optional int64 unk11 = 13;
}

message BSSIDGeo { 
    required string bssid = 1; 
    optional Location location = 2; 
}

message WiFiLocation {
    optional int64 unk1 = 1; 
    repeated BSSIDGeo wifi = 2; 
    optional int32 unk2 = 3; 
    optional int32 single = 4; 
    optional string APIName = 5;
}
